llama-cli/Cargo.toml

```diff
@@ -11,12 +11,13 @@ llama-rs = { path = "../llama-rs", features = ["convert", "quantize"] }
 rand = {workspace = true}
 
 bincode = "1.3.3"
+bytesize = "1.1"
 clap = {version = "4.1.8", features = ["derive"]}
 color-eyre = {version = "0.6.2", default-features = false}
 env_logger = "0.10.0"
 log = "0.4"
 num_cpus = "1.15.0"
 once_cell = "1.17.1"
 rustyline = "11.0.0"
-spinners = "4.1.0"
+spinoff = { version = "0.7.0", default-features = false, features = ["dots2"] }
```

Comment:
These are pretty similar, but `spinoff` is much more versatile as it allows updating the message